nomad/fsm_test.go

```diff
@@ -2879,7 +2877,10 @@ func TestFSM_ReconcileVersionedSummary(t *testing.T) {
 	}
 
 	require.NoError(t, state.UpsertJob(structs.MsgTypeTestSetup, 1000, nil, job1))
-	require.NoError(t, state.UpsertAllocs(structs.MsgTypeTestSetup, 1011, []*structs.Allocation{alloc}))
+	require.NoError(t, state.UpsertAllocs(structs.MsgTypeTestSetup, 1011, []*structs.Allocation{alloc1}))
+
+	require.NoError(t, state.DeleteJobSummary(1030, job1.Namespace, job1.ID))
+	require.NoError(t, state.DeleteJobSummary(1040, alloc1.Namespace, alloc1.Job.ID))
```

Comment:
these seem like 2 ways of saying the same thing?

Reply:
i think these indexes are higher than ones used below; they should be increasing

Reply:
Fixed and fixed, thx